etl/steps/data/garden/ggdc/2022-11-28/penn_world_table.py

```diff
@@ -0,0 +1,76 @@
+import json
+from typing import List, cast
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+from owid.catalog.utils import underscore_table
+from owid.datautils import geo
+from structlog import get_logger
+
+from etl.helpers import Names
+from etl.paths import DATA_DIR
+
+log = get_logger()
+
+# naming conventions
+N = Names(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("penn_world_table.start")
+
+    # read dataset from meadow
+    ds_meadow = Dataset(DATA_DIR / "meadow/ggdc/2022-11-28/penn_world_table")
+    tb_meadow = ds_meadow["penn_world_table"]
+
+    df = pd.DataFrame(tb_meadow)
+
+    log.info("penn_world_table.exclude_countries")
+    df = exclude_countries(df)
+
+    log.info("penn_world_table.harmonize_countries")
+    df = harmonize_countries(df)
+
+    ds_garden = Dataset.create_empty(dest_dir)
+    ds_garden.metadata = ds_meadow.metadata
+
+    tb_garden = underscore_table(Table(df))
+    tb_garden.metadata = tb_meadow.metadata
+    for col in tb_garden.columns:
+        tb_garden[col].metadata = tb_meadow[col].metadata
+
+    ds_garden.metadata.update_from_yaml(N.metadata_path)
+    tb_garden.update_metadata_from_yaml(N.metadata_path, "penn_world_table")
```

Comment:
Just so you know, this line will read the metadata yaml file, and apply all the metadata under "table", including variable metadata (i.e. variable names, units, etc.).